### PR TITLE
Fixed deprecated imports of H/VGroup in favor of Stack

### PR DESCRIPTION
**Pull request by teodosii**
# What this PR does

Closes https://github.com/grafana/irm/issues/10

### CODE REVIEW

File: grafana-plugin/.eslintrc.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = {
     {
       files: ['src/**/*.{ts,tsx}'],
       rules: {
-        'deprecation/deprecation': 'off',
+        'deprecation/deprecation': 'warn',
```
Comment (brojd) [inline grafana-plugin/.eslintrc.js:15]: isn't it warn by default? If so we can just remove  whole `rules` from here

Comment (teodosii) [reply grafana-plugin/.eslintrc.js:15]: First time I attempt this by removing the rule completely didn't trigger any warnings/errors 🤔

File: .github/workflows/linting-and-tests.yml [hunk 2]
```diff
@@ -242,8 +242,7 @@ jobs:
     strategy:
       matrix:
         grafana_version:
-          - 10.1.7
-          - 10.3.3
+          - 11.1.3
```
Comment (joeyorlando) [inline .github/workflows/linting-and-tests.yml:245]: nice 🔥 

should we also still test against `10.x` versions? wdyt?

**Review by joeyorlando — APPROVED** [review]

File: .github/workflows/linting-and-tests.yml (hunk 2 above)
Comment (teodosii) [reply .github/workflows/linting-and-tests.yml:245]: Not really possible anymore after adding these changes, afaik these UI changes are only available in grafana 11.

Comment (teodosii) [reply .github/workflows/linting-and-tests.yml]: Checked&tested, lowest we can do is `10.2.5` which I have it set in the e2e yaml file

File: grafana-plugin/e2e-tests/outgoingWebhooks/advancedWebhook.test.ts [hunk 3]
```diff
@@ -22,12 +22,15 @@ test('create advanced webhook and check it is displayed on the list correctly',
...
   await webhooksFormDivs.locator('[name=name]').fill(WEBHOOK_NAME);
 
-  // Select team
-  await page.getByLabel('New Outgoing Webhook').getByRole('img').nth(1).click(); // Open team dropdown
-  await page.getByLabel('Select options menu').getByText('No team').click(); // Select "No team"
+  // Open team dropdown
+  await page.getByTestId('team-selector').locator('div').filter({ hasText: 'Choose (Optional)' }).nth(1).click();
+  // Set No Team
+  await page.getByLabel('Select options menu').getByText('No team').click();
+  // await page.getByTestId('data-testid Select option').getByText('No team').click();
```
Comment (brojd) [inline grafana-plugin/e2e-tests/outgoingWebhooks/advancedWebhook.test.ts:29]: Can we remove all those comments with the old ways?

**Review by brojd — APPROVED** [review]

**Pull request merged**